ansible-advancedlinux/README.md

```diff
@@ -27,9 +28,32 @@ This template also ilustrates how to use Outputs and Tags.
  * The template will generate an output with the fqdn of the new public IP so you can easily connect to the Ansible VM.
  * The template will associate two tags to all the VMS : ServerRole (Webserver,database etc) and ServerEnvironment (DEV,PRE,INT, PRO etc)
 
+Below are the parameters that the template expects
```

Comment:
Don't need this table anymore. Parameters and description are rendered on Azure.com index